tests/unit_test.py

```diff
@@ -19,6 +19,7 @@
 import os
 import re
 import uuid
+from io import StringIO
```

Comment:
```suggestion

```
This file appears to have several unused imports. Would we want to remove those?

Reply:
Good catch thanks! - I went ahead and removed all unused imports and variables from that file, and re-organised them.